app/assets/javascripts/application.js

```diff
@@ -13,4 +13,6 @@
 //= require rails-ujs
 //= require activestorage
 //= require turbolinks
+//= require jquery3
```

Comment:
I think that requiring jquery3 here won't make a difference since `application.js` is not being included in any file.
What do you think about including it in the main layout so this and other libraries like `rails-ujs` and others start to be included in the webpage?

Reply:
I ended up adding <%= javascript_include_tag 'application', 'data-turbolinks-track': 'reload' %> again to application.html.erb as I explained in https://github.com/anita040593/RetroApp/pull/5 for this new version :)